sphinx/ext/autodoc/__init__.py

```diff
@@ -269,12 +269,13 @@ def documenters(self) -> Dict[str, "Type[Documenter]"]:
         """Returns registered Documenter classes"""
         return self.env.app.registry.documenters
 
-    def add_line(self, line: str, source: str, *lineno: int) -> None:
+    def add_line(self, text: str, source: str, *lineno: int) -> None:
```

Comment:
I don't think this change makes sense, given the function name. Perhaps add an extra `add_text` function that loops over each line and calls add_line?

Reply:
I have no opinion. So +0 for adding `add_text()`.